BottomBar.Droid/Renderers/BottomBarPageRenderer.cs

```diff
@@ -227,7 +231,8 @@ void UpdateBarTextColor ()
 				return;
 			}
 
-			// haven't found yet how to set text color for tab items on_bottomBar, doesn't seem to have a direct way
+            _bottomBar.SetActiveTabColor(Element.BarTextColor.ToAndroid());

```

Comment:
This is my first PR to this project and I felt you comments quite harsh as if I'm an idiot who cannot format his code before pushing.